controllers/clusterresources/awsvpcpeering_controller.go

```diff
@@ -49,6 +48,8 @@ type AWSVPCPeeringReconciler struct {
 	EventRecorder record.EventRecorder
 }
 
+const awsVPCPeeringStatusCodeDeleted = "deleted"
```

Comment:
Let's use const from package models instead of this variable

Reply:
Moved to the models pkg